hugolib/filesystems/basefs.go

```diff
@@ -55,6 +55,9 @@ type BaseFs struct {
 	// This usually maps to /my-project/public.
 	PublishFs afero.Fs
 
```

Comment:
All the code below needs to be reverted. If I remember correctly, there is no static files copied in the regular site building process.

Reply:
Or is this used for the files that gets copied only? I don't see that distinction in the code below.

Reply:
> Or is this used for the files that gets copied only?

Yes. This is used [here](https://github.com/gohugoio/hugo/blob/59d3913e9af6d99481e3a8fb82ce4122ae8c6ead/hugolib/pages_process.go#L162)

If not, static files in `contentDir` are copied to memory. This will not be what we want.

(I fixed the name)